test/NuGet.Core.FuncTests/Dotnet.Integration.Test/PackCommandTests.cs

```diff
@@ -0,0 +1,392 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using NuGet.Frameworks;
+using NuGet.Packaging;
+using NuGet.Test.Utility;
+using NuGet.Versioning;
+using NuGet.XPlat.FuncTest;
+using Xunit;
+
+
+namespace NuGet.Commands.FuncTest
+{
+    [Collection("Dotnet Integration Tests")]
```

Comment:
I thought classes just extended the fixtures? Aren't collections more for grouping how tests run in parallel? (I don't know a lot about this part of xunit)